docs/releases/3.1.txt

```diff
@@ -96,6 +96,25 @@ and generate and apply a database migration. For now, the old fields and
 transforms are left as a reference to the new ones and are :ref:`deprecated as
 of this release <deprecated-jsonfield>`.
 
+.. _default-hashing-algorithm-usage:
+
+``DEFAULT_HASHING_ALGORITHM`` settings
+--------------------------------------
+
+The new :setting:`DEFAULT_HASHING_ALGORITHM` transitional setting allows
+defining the default hashing algorithm to use for encoding cookies, password
+reset tokens in the admin site, user sessions, and signatures created by
+:class:`django.core.signing.Signer` and :meth:`django.core.signing.dumps`.
+
+If you upgrade multiple instances of the same app to Django 3.1, you should
+set :setting:`DEFAULT_HASHING_ALGORITHM` to ``'sha1'``, because hashes that use
+SHA-1 algorithm are valid in Django 3.1 and in older versions of Django. You
```

Comment:
Just this one line that I'm not sure on.

Pre-coffee sketch:

> Support for SHA-256 was added in Django 3.1. If you are uprading multiple instances of the same app to Django 3.1, you should set ... to `'sha1'` during the transition, in order to allow compability with the older versions of Django. Once the transition to 3.1 is complete...

Reply:
Should it be "project" rather than "app"?

Reply:
Probably "pre-coffee"

Reply:
It's not you, it was in the original too… 😉☕

Reply:
Updated.